tracer/test/Datadog.Trace.Debugger.IntegrationTests/Helpers/DebuggerTestHelper.cs

```diff
@@ -94,6 +93,13 @@ internal static (ProbeAttributeBase ProbeTestData, SnapshotProbe Probe)[] GetAll
         return snapshotLineProbes.Concat(snapshotMethodProbes).ToArray();
     }
 
+    private static bool IsOptimized(Assembly assembly)
+    {
+        var debuggableAttribute = (DebuggableAttribute)Attribute.GetCustomAttribute(assembly, typeof(DebuggableAttribute));
```

Comment:
`GetCustomAttribute` might return `null` if the attribute does not exist.

Reply:
You have a point here, but I don't think it can happen in our case 🤔